packages/block-library/src/image/style.scss

```diff
@@ -273,6 +317,32 @@
 	}
 }
 
-html.has-lightbox-open {
-	overflow: hidden;
+@keyframes lightbox-zoom-in {
+	0% {
+		left: var(--lightbox-initial-left-position);
+		top: var(--lightbox-initial-top-position);
+		transform: scale(var(--lightbox-scale-width), var(--lightbox-scale-height));
+	}
+	100% {
+		left: var(--lightbox-target-left-position);
+		top: var(--lightbox-target-top-position);
+		transform: scale(1, 1);
+	}
+}
+
+@keyframes lightbox-zoom-out {
+	0% {
+		visibility: visible;
+		left: var(--lightbox-target-left-position);
```

Comment:
👋  Just dropping by and skimming through the code, and noticed that you may be animating the position of the lightbox via `top` and `left` CSS properties — have you considered using `transform` (or the new `translate` prop) [for more performant animations](https://web.dev/animations-guide/)?

Reply:
Great, thanks for the suggestion! Will take a look